src/Standards/BestIt/Sniffs/Commenting/AbstractDocSniff.php

```diff
@@ -66,9 +66,9 @@ abstract class AbstractDocSniff extends AbstractSniff
     /**
      * The cached position of the summary.
      *
-     * @var int|null
+     * @var int|bool
      */
-    private $summaryPosition = -1;
+    private $summaryPosition = false;
```

Comment:
As I understand it the purpose is to use false as an initial-value, like a helper. I dislike this use of "false" as helper for "first-use" or "undefined", whatever... Strictly speaking you have a boolean, but you only handle "false" and do not handle "true"-values. So imagine summaryPosition gets "true" then your logic breaks, even the getter-function below in line 125, which returns true, though it should only return null/int. What's the reason for not using null|int instead? Just curious... :)

Reply:
Updated PR, exchanged false with null on all occasions i could find, tests run successfully so I think this works as well.

Reply:
I don't like this. We shouldn't introduce peroperties that can be multiple types. I see the problem that null is a valid value for the cached position and null indicates also that the cache is not filled. Without a proper caching implementation via PSR-6 i don't see a clean solution. So its okay for me